dref/views.py

```diff
@@ -0,0 +1,152 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+from rest_framework import (
+    views,
+    viewsets,
+    response,
+    permissions,
+    status,
+    mixins
+)
+from rest_framework.decorators import action
+from dref.models import (
+    Dref,
+    NationalSocietyAction,
+    PlannedIntervention,
+    IdentifiedNeed,
+    DrefFile,
+)
+from dref.serializers import (
+    DrefSerializer,
+    NationalSocietyActionSerializer,
+    PlannedInterventionSerializer,
+    IdentifiedNeedSerializer,
+    DrefFileSerializer,
+    IdentifiedNeedSerializer,
+)
+from dref.filter_set import DrefFilter
+
+
+class DrefViewSet(viewsets.ModelViewSet):
+    # TODO: Need to add permission to delete?
```

Comment:
Is this done?

Reply:
anyone can delete, who have access to dref.

Reply:
okay